servers/rest-services/src/main/java/org/projectnessie/services/rest/RestTreeResource.java

```diff
@@ -65,8 +71,41 @@ private TreeService resource() {
   @JsonView(Views.V1.class)
   @Override
   public ReferencesResponse getAllReferences(ReferencesParams params) {
-    Preconditions.checkArgument(params.pageToken() == null, "Paging not supported");
```

Comment:
Why remove this now? Paging will not work for references even with this PR, will it?

Reply:
V1 doesn't return a paging token, so there's nothing that could pass it in.